tests/kernel/run_ltp.pm

```diff
@@ -262,6 +262,27 @@ sub run {
     script_run('$LTPROOT/ver_linux');
 
     if ($is_network) {
+        # poo#18762: Sometimes there is physical NIC which is not configured.
+        # One of the reasons can be renaming by udev rule in
+        # /etc/udev/rules.d/70-persistent-net.rules. This breaks some tests
+        # (even net namespace based ones).
+        # Workaround: configure physical NIS (if needed).
+        my $tmp = << 'EOF';
+dir=/sys/class/net
+ifaces="`basename -a $dir/* | grep -v -e ^lo -e ^tun -e ^virbr -e ^vnet`"
+for iface in $ifaces; do
+    config=/etc/sysconfig/network/ifcfg-$iface
+    if [ "`cat $dir/$iface/operstate`" = "down" ] && [ ! -e $config ]; then
+        echo "WARNING: create config '$config'"
+        printf "BOOTPROTO='dhcp'\nSTARTMODE='auto'\nDHCLIENT_SET_DEFAULT_ROUTE='yes'\n" > $config
+        systemctl restart network
+        sleep 1
+    fi
+done
+EOF
+        chomp $tmp;
+        script_run($tmp);
```

Comment:
what is happening here I wonder:
```
  systemctl stop NetworkManager SuSEfirewall2 wicked wickedd ; echo vDfOE-$?-
  6_Ju6-0-
  # systemctl stop NetworkManager SuSEfirewall2 wicked wickedd ; echo vDfOE-$?-
```
Maybe you need to execute that script block with `script_output` or whatever it is called.